### ISSUE TITLE
Bug: First Super-User account is not protected

### ISSUE THREAD

**Issue by DevPhilB** [conclurer/midnightboard#243]
**Describe the bug**
As a Super-User I can delete my account in the CMS and I'm still in "logged-in" state.

**To Reproduce**
1. Log-in as Super-User
2. Go to /cms
3. Delete your account

**Expected behavior**
Account of the first Super-User has to be protected and undeletable!

**Comment by wasdJens** [comment]
To add to this maybe: All `users` that can delete other `users` should not be able to delete itself. To achieve this you can check who made the request(Dont know your jwt structure but you can add info like this to the jwt aswell) and compare with the user that is about to be deleted (user delete id same as the one in the jwt? dont delete send an error). 

Some edge-cases thoughts or possible user requirements
> What if I want to remove myself from a company?

Then someone else does this 😅 

> I want to delete the last super user and all other users

Basically a system wipe - just delete the whole system in this case 😅
